### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by y-yu**
- Qulacs build and test phases are turning complicated.
    - `make`, `mate test`, `make pythontest`, `python setup.py install` etc...
- So I add `CONTRIBUTING.md` which says the way how to bulid, format (#201 ) and test.
- I want to discuss if other things are needed to be written in `CONTRIBUTING.md` or not 🤔

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+$ ./script/build_msvc_2015.bat
+$ python setup.py install
+$ cmake --build ./visualstudio --target test --config Release
+$ cmake --build ./visualstudio --target pythontest --config Release
+```
+  
+### Tips
+
+- We only check the format in Linux
+  so Travis CI could point out some format errors even if there is nothing error in your Windows or macOS.
```
Comment (TenninYan) [inline CONTRIBUTING.md]: We only check the format in Linux.
Travis CI could point out format errors even if there are no errors in your Windows or macOS.

が正しい英語かなー？

Comment (y-yu) [reply CONTRIBUTING.md]: Would it be better to use `there is no error` instead of `there are no errors`?

Comment (TenninYan) [reply CONTRIBUTING.md]: そう変更してください！

Comment (y-yu) [reply CONTRIBUTING.md]: Sorry for my very very late fix 🙇

**Comment by corryvrequan** [comment]
LGTM!

**Pull request merged**